cli/ioctl/cmd/account/accountimport.go

```diff
@@ -85,15 +83,11 @@ func writeToFile(alias, addr string) (string, error) {
 		alias), nil
 }
 func readPasswordFromStdin() (string, error) {
-	passwordBytes, err := terminal.ReadPassword(int(syscall.Stdin))
+	password, err := util.ReadSecretFromStdin()
 	if err != nil {
 		log.L().Error("failed to get password", zap.Error(err))
 		return "", err
 	}
-	password := strings.TrimSpace(string(passwordBytes))
```

Comment:
as we discussed, we need to verify the length of the private key in a follow up PR.